docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,130 @@
+---
+audience: administrator
+components:
+  - serving
+  - eventing
+function: reference
+---
+# Overview
+
+This page provides guidance for administrators on how to install and manage Knative on an existing Kubernetes cluster, and assumes you have familiarity the following:
+
+- Kubernetes and Kubernetes administration.
+- The `kubectl`CLI tool. You can use existing Kubernetes management tools (policy, quota, etc) to manage Knative workloads.
+- The Cloud Native Computing Foundation (CNCF) for which Knative is one of its projects, along with Kubernetes, Prometheus, and Istio.
```

Comment:
```suggestion
- The Cloud Native Computing Foundation (CNCF) an advocacy and support organization for open-source cloud software projects including Knative, Kubernetes, Prometheus, and Istio.
```

Why does the user need to be familiar with CNCF? Does that affect the administration tasks in any way? If not, I'd remove this requirement. CNCF ownership of Knative should be discussed elsewhere, for example in governance, support, and communications.

If this does need to be included here, add a link to the CNCF website and/or to the CNCF project pages for these projects.